rs/registry/admin/src/main.rs

```diff
@@ -5995,41 +5959,33 @@ impl GovernanceCanisterClient {
         decode_make_proposal_response(response)
     }
 
-    pub async fn submit_install_code_proposal(
+    pub async fn submit_external_proposal_candid<T: CandidType>(
```

Comment:
NCR

Can you move this after submit_proposal_action? I think it would make the diff easier to read.

Reply:
Hmm... on my side the diff isn't rendered this way:

![image](https://github.com/user-attachments/assets/30dd07e3-c9f4-4691-b6c1-f96aa054c118)

Perhaps it's not a good idea to let the way github renders diff (which doesn't seem to be consistent by itself) dictate where to add a piece of code...